ci-operator/step-registry/cucushift/installer/rehearse/azure/ipi/disconnected/private/cco-manual-workload-identity/deprovision/OWNERS

```diff
@@ -0,0 +1,16 @@
+approvers:
+- jianlinliu
+- Amoghrd
+- yunjiang29
+- mgahagan73
```

Comment:
Here you can remove `Amoghrd` `mgahagan73`, and replace `MayXuQQ` with @jinyunma

Reply:
got it, thanks.